### DISCUSSION TITLE
Define what to show up in the Searchpane

### DISCUSSION THREAD

**Discussion by suneworld** [EvotecIT/PSWriteHTML#457] [Q&A]
I am creating a Table that has numerous properties. The searchpane shows all these properties, and there are too many. Can I define what properties will show up in the Searchpane?

`New-HTML -Online -TitleText "Test" -FilePath $Filepath {
    New-HTMLTable -DataTable $bigdata -SearchPane
} -Show
`

**Comment by PrzemyslawKlys** [comment]
It's not definable. One couldd probably implement it:
- https://datatables.net/extensions/searchpanes/examples/customFiltering/customPreSelect.html
